esp/templates/program/modules/studentclassregmodule/catalog.html

```diff
@@ -29,6 +31,26 @@
 const register_from_catalog = {{ register_from_catalog|yesno:"true,false" }};
 const hide_empty_cats = {{ "hide_empty_categories"|getBooleanTag|yesno:"true,false" }};
 </script>
+
+<script> 
```

Comment:
It might be better for future us if you move this javascript into /media/scripts/program/modules/catalog.js. Fun fact: if you have a separate javascript file, when you make changes to it you only need to hard refresh your page to see the effects (no shutting down and restarting your server)!